pkgs/applications/misc/fead/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromSourcehut, python3, help2man }:
+
+stdenv.mkDerivation rec {
+  pname = "fead";
+  version = "0.1.2";
+
+  src = fetchFromSourcehut {
+    owner = "~cnx";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-prDVBFpR3kx5jM+KY8XWMaXQ4i9A9ytFelqKJJu5j2E=";
+  };
+
+  nativeBuildInputs = [ help2man ];
+  buildInputs = [ python3 ];
+
+  postPatch = ''
+    patchShebangs src/fead.py
+  '';
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  dontPatchShebangs = true;
```

Comment:
Is there a particular reason why we want this?

Reply:
The script is used for generating man pages, hence patching #! post patch, so it's no longer necessary at this stage.

Reply:
Please add this as a short comment.

Reply:
Is the current one good enough?

Reply:
I meant adding a comment to ``dontPatchShebangs = true;``, mostly because it is a bit unusual and the next person should probably not remove it.

Reply:
Ah right, done!